google/resource_bigtable_instance.go

```diff
@@ -44,11 +44,17 @@ func resourceBigtableInstance() *schema.Resource {
 			},
 
 			"num_nodes": {
-				Type:         schema.TypeInt,
+				Type:     schema.TypeInt,
+				Optional: true,
+				ForceNew: true,
```

Comment:
Doing things like that breaks the 1:1 mapping from config -> the API, and imo isn't worth it if we ever need to correct later. For example, if we were to gain the ability to read `num_nodes`, we would either need to conditionally read it or would see diffs in the future.

I think it's worth a breaking change here because the number of users is small (and I expect the # of users that relied on the implicit default is smaller), the change is easy to make, and it means we'll be tracking the API as accurately as we can until the next client update.